src/libexpr/eval.cc

```diff
@@ -364,6 +364,36 @@ EvalState::EvalState(
 
 EvalState::~EvalState()
 {
+  auto profileFile = getEnv("NIX_PROFILE_FILE");
```

Comment:
This should be a setting.

EDIT: and so should `NIX_SHOW_STATS` actually

Reply:
Agreed.